packages/types/package.json

```diff
@@ -3,6 +3,10 @@
   "version": "1.0.0-beta.37",
   "description": "Implementation of the Parity codec",
   "main": "index.js",
+  "bin": {
+    "polkadot-types-from-chain": "./scripts/polkadot-types-from-chain.js",
+    "polkadot-types-from-defs": "./scripts/polkadot-types-from-defs.js"
+  },
```

Comment:
Why not put these in api? They are only relevant when using api.

Reply:
I thought to put them there - but since it is all type extraction, left them in types.